pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -65,34 +65,42 @@ func GetPolicyLabels(ns, name string) labels.LabelArray {
 	}
 }
 
+// getEndpointSelector converts the provided labelSelector into an EndpointSelector,
+// adding the relevant matches for namespaces based on the provided options.
+func getEndpointSelector(namespace string, labelSelector *metav1.LabelSelector, addK8sPrefix, matchesInit bool) api.EndpointSelector {
+	es := api.NewESFromK8sLabelSelector("", labelSelector)
+
+	// There's no need to prefixed K8s
```

Comment:
I don't understand what this sentence means?! Maybe:
```
// The k8s prefix must not be added to reserved labels.
```

Reply:
I'll hold off on this for a bunch of other minor cosmetic touchups.